examples/asr/emformer_rnnt/README.md

```diff
@@ -63,3 +63,25 @@ The table below contains WER results for dev and test subsets of TED-LIUM releas
 | test        |       0.098  |
 
 [`tedlium3/eval_pipeline.py`](./tedlium3/eval_pipeline.py) evaluates the pre-trained `EMFORMER_RNNT_BASE_TEDLIUM3` bundle on the dev and test sets of TED-LIUM release 3. Running the script should produce WER results that are identical to those in the above table.
+
+### MuST-C release v2.0
+
+The MuST-C model is configured with a vocabulary size of 500. Consequently, the MuST-C model's last linear layer in the joiner has an output dimension of 501 (500 + 1 to account for the blank symbol). Different from the above two models, we keep the transcriptions cased and punctuated when training the SentencePiece model.
```

Comment:
the current description seems to suggest that we chose to not keep transcriptions cased and punctuated for the other two models, when in actuality we didn't have a choice because the datasets themselves aren't cased and punctuated. we should clarify this, e.g. something like

```suggestion
The MuST-C model is configured with a vocabulary size of 500. Consequently, the MuST-C model's last linear layer in the joiner has an output dimension of 501 (500 + 1 to account for the blank symbol). In contrast to those of the datasets for the above two models, MuST-C's transcripts are cased and punctuated; we preserve the casing and punctuation when training the SentencePiece model.
```